src/main/java/de/dagere/peass/ci/MeasurementOverviewAction.java

```diff
@@ -88,16 +88,16 @@ public Changes getChanges() {
    }
 
    public boolean testIsChanged(final String testcase) {
-      boolean isChanged = false;
+      final boolean isChanged = true;
       for (Entry<String, List<Change>> changeEntry : changes.getTestcaseChanges().entrySet()) {
          for (Change change : changeEntry.getValue()) {
             final String changedTestcase = changeEntry.getKey() + "#" + change.getMethod();
             if (testcase.equals(changedTestcase)) {
-               isChanged = true;
```

Comment:
The goal of this implementation is to have single point of exit. Therefore, I'd leave it as it is.

Additionally, this refactoring does not make sense: Declaring a final boolean in the beginning and returning it in the end does not make sense, in this case, only the direct value should be returned (but than, we don't have a single point of exit, which makes debugging harder, therefore, I'd try do not do this).